coalib/misc/Constants.py

```diff
@@ -109,7 +110,16 @@
                                     'languages',
                                     'definitions')
 
-system_coafile = os.path.join(coalib_root, 'default_coafile')
+# system_coafile = os.path.join(coalib_root, 'system_coafile')
+for file_lists in os.listdir(coalib_root):
+    for file in file_lists:
+        if file == 'default_coafile':
+            system_coafile = os.path.join(coalib_root, 'default_coafile')
+        else:
+            system_coafile = os.path.join(coalib_root, 'system_coafile')
+
```

Comment:
You can use a more compact logic to achieve the above result.
file = [i for i in os.listdir(coalib_root) if "coafile" in i][0]
system_coafile = os.path.join(coalib_root, file)